src/redist/targets/GenerateMSIs.targets

```diff
@@ -324,9 +324,12 @@
     </ItemGroup>
     <PropertyGroup>
       <LatestTemplateMsiInstallerFile>@(LatestTemplateInstallerComponent->'%(MSIInstallerFile)')</LatestTemplateMsiInstallerFile>
+
+      <UpgradePoliciesSrcPath>$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle\upgradePolicies.wxs</UpgradePoliciesSrcPath>
```

Comment:
Does the package reference have `GeneratePathProperty="true"` to allow for `PkgMicrosoft_DotNet_Build_Tasks_Installers` to exist?

Reply:
yes